includes/Framework/Helper.php

```diff
@@ -406,7 +406,7 @@ public static function is_rest_api_request() {
 	 */
 	public static function trigger_error( $message, $type = E_USER_NOTICE ) {
 
-		if ( is_callable( 'is_ajax' ) && is_ajax() ) {
```

Comment:
💅🏻 Just curious if `is_callable` is still necessary here. Facebook for WooCommerce [requires WP 5.6+](https://github.com/woocommerce/facebook-for-woocommerce/blob/1e2d5407b24114706fe9eb8374b5f9e490c00dbf/facebook-for-woocommerce.php#L15) and `wp_doing_ajax` has been around since [4.7.0](https://github.com/WordPress/wordpress-develop/blob/a8fd670237e4983b08a2f92d492946d570d8238b/src/wp-includes/load.php#L1531-L1540). (Although it looks like `Helper::trigger_error` isn't actually used anywhere in the extension).

Reply:
Yes. Good catch, @layoutd. I removed trigger_error altogether.

 It looks like the whole class was ported from SkyVerge. Only a few of the methods in the class are used. I created https://github.com/woocommerce/facebook-for-woocommerce/issues/2480 to clean this up.